webapp/channels/src/components/channel_header_dropdown/menu_items/toggle_mute_channel/toggle_mute_channel.tsx

```diff
@@ -43,42 +43,36 @@ type Props = {
     actions: Actions;
 };
 
-export default class MenuItemToggleMuteChannel extends React.PureComponent<Props> {
-    handleClick = () => {
-        const {
-            user,
-            channel,
-            isMuted,
-            actions: {
-                updateChannelNotifyProps,
-            },
-        } = this.props;
+export default function MenuItemToggleMuteChannel({
+    id,
+    isMuted,
+    channel,
+    user,
+    actions}: Props) {
```

Comment:
Small nit, just to be the same in all the codebase (yes, I should find a linting rule for this :sweat_smile: )
```suggestion
    actions,
}: Props) {
```